symfony/src/Controller/PatientStatusController.php

```diff
@@ -0,0 +1,176 @@
+                    $patientStatusHistory->setImport($patientStatusImport);
+                    $em->persist($patientStatusHistory);
+
+                    // Update history id in patient_status table
+                    $patientStatus->setHistory($patientStatusHistory);
+                    if (($key % $batchSize) === 0) {
+                        $em->flush();
+                        $loggerService->log(Log::PATIENT_STATUS_HISTORY_ADD, $patientStatusHistory->getId());
+                        $loggerService->log(Log::PATIENT_STATUS_EDIT, $patientStatus->getId());
+                        $em->clear(PatientStatusHistory::class);
+                        $em->clear(PatientStatus::class);
+                    }
+                }
+                $em->flush();
+                // Update confirm status
+                $patientStatusImport->setConfirm(1);
+                $em->flush();
+                $loggerService->log(Log::PATIENT_STATUS_IMPORT_EDIT, $patientStatusImport->getId());
+                $em->clear();
+                $this->addFlash(
+                    'success',
+                    'Successfully Imported!'
+                );
+            } else {
+                $this->addFlash(
+                    'notice',
+                    'Import status canceled!'
+                );
+                $em->remove($patientStatusImport);
+                $em->flush();
+            }
+            return $this->redirectToRoute('patientStatusImport');
+        }
+        return $this->render('patientstatus/confirmation.html.twig', [
+            'patientStatuses' => $importPatientStatuses,
+            'importConfirmForm' => $form->createView(),
+            'rowsCount' => count($patientStatusImport->getPatientStatusTemps())
+        ]);
+    }
+
+    /**
+     * @Route("/patient/status/import/{id}", name="patientStatusImportDetails", methods={"GET", "POST"})
+     */
+    public function patientStatusImportDetails(int $id, Request $request, EntityManagerInterface $em, PatientStatusImportService $patientStatusImportService)
+    {
+        $patientStatusImport = $em->getRepository(PatientStatusImport::class)->findOneBy(['id' => $id, 'user_id' => $this->getUser()->getId(), 'confirm' => 1]);
+        if (empty($patientStatusImport)) {
+            throw $this->createNotFoundException('Page Not Found!');
+        }
+        //For ajax requests
+        if ($request->isXmlHttpRequest()) {
+            $params = $request->request->all();
+            $patientStatusHistories = $patientStatusImport->getPatientStatusHistories()->slice($params['start'], $params['length']);
+            $ajaxData = [];
+            $ajaxData['data'] = $patientStatusImportService->getAjaxData($patientStatusHistories, $patientStatusImport->getOrganization());
+            $ajaxData['recordsTotal'] = $ajaxData['recordsFiltered'] = count($patientStatusImport->getPatientStatusHistories());
+            return new JsonResponse($ajaxData, 200);
```

Comment:
Default response code is 200. Not sure of the value of explicitly setting it here.